app/models/protocols_participant.rb

```diff
@@ -85,7 +84,7 @@ def update_appointments_on_arm_change
   end
 
   def can_be_destroyed?
-    procedures.where.not(status: 'unstarted').empty?
+    self.can_be_destroyed
```

Comment:
Pretty sure you can just remove this method.  Rails automatically creates ? methods for boolean attributes.  Please test this and remove the method if possible.